taichi/ir/ir.h

```diff
@@ -1552,5 +1552,4 @@ inline void StmtFieldManager::operator()(const char *key, T &&value) {
 
 TLANG_NAMESPACE_END
 
-#include "taichi/ir/statements.h"
 #include "taichi/ir/visitors.h"
```

Comment:
What about also remove this? We need to include `visitors.h` in some but not all files that include `ir.h`.